src/test-apps/TestWeaveTunnelBR.cpp

```diff
@@ -1107,7 +1107,7 @@ static void TestTunnelNoStatusReportResetReconnectBackoff(nlTestSuite *inSuite,
     gTestSucceeded = false;
     /* Set the test timeout to be a little longer than the Tunnel Control ExchangeContext
      * timeout */
-    gMaxTestDurationMillisecs = (WEAVE_CONFIG_TUNNELING_CTRL_RESPONSE_TIMEOUT_SECS + 1 ) * System::kTimerFactor_milli_per_unit;
+    gMaxTestDurationMillisecs = (WEAVE_CONFIG_TUNNELING_CTRL_RESPONSE_TIMEOUT_SECS + 5 ) * System::kTimerFactor_milli_per_unit;
```

Comment:
maybe use some macro naming to represent the 1 or 5,

Reply:
This time was just a guard time for the test completion and so I left it hardcoded. Increased it slightly because I noticed the test sometimes taking slightly longer.